python/paddle/v2/framework/tests/test_rank_loss_op.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+class TestReshapeOp(OpTest):
+    def setUp(self):
+        self.op_type = "rank_loss"
+        num = 5
+        # P = {0, 1.0} or {0, 0.5, 1.0}
+        P = np.random.randint(0, 2, size=(num, num)).astype("float32")
+        Oi = np.random.random((num, num)).astype("float32")
```

Comment:
Local variable  name should be `lower_with_under`, https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming

Reply:
Done